packages/keycodes/src/platform.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Return true if platform is MacOS.
+ *
+ * @param {Object} _window   window object by default; used for DI testing.
+ *
+ * @return {boolean}         True if MacOS; false otherwise.
+ */
+export function isAppleOS( _window = window ) {
```

Comment:
Can we keep the original name?

We can re-export it instead of aliasing:
```js
export const isMacOS = isAppleOS;`
// vs
export * from './platform';
```

Reply:
I'm a bit confused, do we to keep the function name `isAppleOS` and only add an extra export with the name `isMacOS`?

Reply:
I'm not sure why we needed `isAppleOS` in the first place. If we can remove it then sure.